src/components/Tab/Tabs.tsx

```diff
@@ -12,6 +12,10 @@ export interface FlowbiteTabTheme {
   tablist: {
     base: string;
     styles: TabStyles;
+    tabitemcontainer: {
```

Comment:
I would say that probably we want to rename this one to `tabItemContainer`.

Reply:
I followed the naming of the other components, they are respectively called: tabitem and tablists. So having only this one as camel case is probably going to confuse everyone. We could change every key to a camel case one, but it probably need a separate PR.